test/Validate.SSA.cpp

```diff
@@ -75,7 +74,7 @@ TEST_F(Validate, IdUndefinedBad) {
     )";
   CompileSuccessfully(str);
   ASSERT_EQ(SPV_ERROR_INVALID_ID, ValidateInstructions());
-  ASSERT_TRUE(ContainsString(getDiagnosticString(), "missing"));
+  ASSERT_THAT(getDiagnosticString(), HasSubstr("missing"));
```

Comment:
I would switch these ASSERT_THAT to EXPECT_THAT since this is not something that would invalidate any new expectations we add to the test.